tests/Settings.json5

```diff
@@ -1,20 +1,22 @@
 {
-  // c
-  /* c */
-  debug: true,
-  production: false,
-  arr: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10,],
-  place: {
-    name: 'Torre di Pisa',
-    longitude: 43.7224985,
-    latitude: 10.3970522,
-    favorite: false,
-    reviews: 3866,
-    rating: 4.5,
-    creator: {
-      name: "John Smith",
-      "username": "jsmith",
-      "email": "jsmith@localhost",
-    }
-  }
+    // c
+    /* c */
+    debug: true,
+    production: false,
+    arr: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+    place: {
+        name: "Torre di Pisa",
+        longitude: 43.7224985,
+        latitude: 10.3970522,
+        favorite: false,
+        reviews: 3866,
+        rating: 4.5,
+        creator: {
+            name: "John Smith",
+            username: "jsmith",
+            email: "jsmith@localhost",
+        },
+    },
```

Comment:
Same as above.

Reply:
there two more line that I have added to this file 
` "FOO": "FOO should be overridden",
    "bar": "I am bar"
 `
the extra space is introduced by running rustfmt